### ISSUE TITLE
Inconstant user-agent

### ISSUE THREAD

**Issue by fufroma** [icefields/Power-Ampache-2#123]
User-agent is inconstant:
```
REDACTED_IP - - [02/Jun/2024:14:58:21 +0200] "GET /server/json.server.php?action=ping&auth=redacted HTTP/1.1" 200 4566 "-" "PowerAmpache2-1.00-58-free"
REDACTED_IP - - [02/Jun/2024:14:58:23 +0200] "GET /image.php?object_id=2188&object_type=album HTTP/1.1" 200 32343 "-" "okhttp/5.0.0-alpha.9"
REDACTED_IP - - [02/Jun/2024:14:58:23 +0200] "GET /image.php?object_id=9644&object_type=album HTTP/1.1" 200 33910 "-" "okhttp/5.0.0-alpha.9"
REDACTED_IP - - [02/Jun/2024:14:58:23 +0200] "GET /image.php?object_id=5296&object_type=album HTTP/1.1" 200 108955 "-" "okhttp/5.0.0-alpha.9"
```

**Comment by icefields** [comment]
> User-agent is inconstant:
> 

I use a library called Coil to download images, it's one of the most standard libraries for android for Jetpack Compose. I'm not sure I can change the user agent for that, but I will look into it.

**Comment by fufroma** [comment]
https://github.com/coil-kt/coil/blob/main/docs/recipes.md#headers

Look like you have to set manually the User-Agent header

**Comment by icefields** [comment]
yes, I'm already looking into the Interceptor. Probably the patch will be in the next-next release, because I was alredy planning on making a release this evening. Unless I have time to add it today. I'll keep you posted

**Comment by fufroma** [comment]
`okhttp/5.0.0-alpha.9` still showing in `PowerAmpache2-1.00-60-free`

**Comment by fufroma** [comment]
`okhttp/5.0.0-alpha.9` still here in `1.00-61-free`

**Comment by icefields** [comment]
I haven't worked on the agent for images, only for regular requests so far. You'll know when I make this fix becaus I'll put it in the changelog + message in this ticket

**Comment by icefields** [comment]
can you check https://github.com/icefields/Power-Ampache-2/releases/tag/v1.01-75 ? Soon available on Fdroid and PlayStore as well. All user agents should be now fixed
